### PR TITLE
Removed grammatical mistakes from  paragraph 'Here is where regular e…

### PR DESCRIPTION
**Pull request by Doado**
…xpressions'. 

Made some minor stylistic changes and also changed bold font to italic font (admittedly a matter of taste).

### CODE REVIEW

File: regular-expression.dd [hunk 1]
```diff
@@ -13,13 +13,13 @@ $(D_S Regular Expressions,
...
-     put together with a substitution mechanism, they form a Swiss Army knife of text processing.
-     They are considered so useful that a number of languages provides built-in support
-     for regular expressions. This does $(B not) nessary mean that
-     built-in implies $(B faster) processing or more features. It's more a matter of providing
-     $(B convenient and friendly syntax) for typical operations and usage patterns.
+    $(P Here is where $(LUCKY regular expressions) – often just called $(I regexes) – come in handy.
+     Regular expressions are a simple yet comprehensive language for defining string patterns. 
+     Combined with a mechanism for string substitution, regular expressions are a powerful tool for text processing.
+     In fact, they are considered so useful that a number of languages provide built-in support
+     for them. However, built-in support does $(I not) necessarily imply $(I faster) processing
```
Comment (McSherry) [inline regular-expression.dd]: I don't think the italics on this line + lines 22 add anything to the doc. IMO, the emphasis the italics provide makes the sentence read a bit oddly.

**Comment by McSherry** [comment]
Along with my comment on the diff, this should probably be a pull on the Phobos repo ([this file](https://github.com/D-Programming-Language/phobos/blob/master/std/regex/package.d)) so documentation generated from in-code docs are correct.

I imagine that the docs in this repo are autogenerated from the source code, so changing the documentation comments in the Phobos repo will update the docs in here on the next release.

File: regular-expression.dd (hunk 1 above)
Comment (Doado) [inline regular-expression.dd]: Sure, I agree. They don't really add anything.

**Comment by Doado** [comment]
*Along with my comment on the diff, this should probably be a pull on the
Phobos repo (this file
<https://github.com/D-Programming-Language/phobos/blob/master/std/regex/package.d>)
so documentation generated from in-code docs are correct.*

I don't get this bit (yet). I'm new to PRs.

**Comment by McSherry** [comment]
The Phobos repository contains the source code for D's standard library. D supports, in its source code, documentation comments. For example:

```D
/**
 * Multiplies a number by four.
 * 
 * Params:
 *     number = The number to quadruple.
 * 
 * Returns:
 *     The provided number multiplied by four.
*/
auto quadruple(int number)
{
    return number * 4;
}
```

From these in-code comments, you can then generate documentation (like that on the D website, and in this repository). Because the documentation is generated from the source code, the change needs to be made to the source code, otherwise regenerating the documentation would overwrite the change with the old text.

To make the change to the source code, you'll need to pull the [Phobos](https://github.com/D-Programming-Language/phobos/) repository and change the documentation in the `std\regex\package.d` file. Then, when the documentation is next generated from the source code, your change will be there.

**Comment by Doado** [comment]
I understand that the documentation is generated from the Phobos source
files with ddoc. But I don't understand why a change to a page that is not
in Phobos (or any other library doc), in this case
http://dlang.org/regular-expression.html, needs to be pulled in Phobos.

**Comment by CyberShadow** [comment]
> I imagine that the docs in this repo are autogenerated from the source code, so changing the documentation comments in the Phobos repo will update the docs in here on the next release.

No.

This pull request changes:

http://dlang.org/regular-expression.html

This is a dlang.org article that stands separately from the Phobos module documentation:

http://dlang.org/phobos/std_regex.html
http://dlang.org/library/std/regex.html

**Comment by Doado** [comment]
Thanks for clarifying this, Vladimir. I was already wondering.

**Comment by McSherry** [comment]
> No.
>
> This pull request changes:
>
> http://dlang.org/regular-expression.html

Whoops, didn't see that. My mistake, I thought this was a change to the docs rather than an article.

File: regular-expression.dd (hunk 1 above)
Comment (CyberShadow) [inline regular-expression.dd]: If there is consensus, let's remove them and merge?

**Pull request merged**